test/oli_web/live/new_course/course_details_test.exs

```diff
@@ -294,8 +294,8 @@ defmodule OliWeb.NewCourse.CourseDetailsTest do
         "current_step" => 3
       })
 
-      flash = assert_redirect(view, Routes.delivery_path(OliWeb.Endpoint, :index))
-      assert flash["info"] == "Section successfully created."
+      # flash = assert_redirect(view, Routes.delivery_path(OliWeb.Endpoint, :index))
+      # assert flash["info"] == "Section successfully created."
```

Comment:
If the functionality should do a redirect, why do we leave these lines commented out? 

Maybe it would be good to test the redirect that is expected to happen, what do you think?

Reply:
I forgot to uncomment, this test is actually working